src/collective/easyform/interfaces/easyform.py

```diff
@@ -98,19 +98,22 @@ class IEasyForm(Schema):
         defaultFactory=default_resetLabel,
         required=False,
     )
+    directives.write_permission(method=config.EDIT_ADVANCED_PERMISSION)
```

Comment:
Can anyone think of a reason a normal editor would need to set this field? Maybe some AJAX thing? I think it's safe to leave it out.

Reply:
GET are linkable. 
It might have sense.

Reply:
Sounds good. I'll see if I can introduce a new fieldset "Advanced" do de-clutter the "Default" tab from stuff not needed.